src/tutorials/konnector/going-further.md

```diff
@@ -36,7 +36,7 @@ A basic connector workflow involves:
 
 #### Error handling
 
-If your connector hits an issue fetching or saving the data, it can return an error code by throwing it as an error. The error codes are defined inside the [Cozy Home][] application and will display an explicit error to the user:
+If your connector hits an issue fetching or saving the data, it can return an error code by throwing it as an error. The error codes are defined inside [cozy-konnector-libs][] and [Cozy Harvest Lib][https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/locales/en.json#L124-L208] will display an explicit error to the user:
```

Comment:
```patch
- [cozy-konnector-libs][]
+ [cozy-konnector-libs][https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/src/helpers/errors.js]
```

Reply:
already defined in the end of the file ;). This is not needed.

Reply:
Oh right, I missed that, sorry.
I just noticed the other link is broken

```patch
- [][]
+ []()

Reply:
And probably using an [anchor link](https://gist.github.com/rachelhyman/b1f109155c9dafffe618) would be better